<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>634</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>627</width>
    <height>634</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="toolBoxPage1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>591</width>
        <height>418</height>
       </rect>
      </property>
      <attribute name="label">
       <string>General</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="saveBeforeExecuteCheckBox_">
         <property name="text">
          <string>Auto-save before execute</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>591</width>
        <height>418</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Network Editor</string>
      </attribute>
      <widget class="QGroupBox" name="layoutOptionsGroupBox_">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>531</width>
         <height>291</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>531</width>
         <height>231</height>
        </size>
       </property>
       <property name="title">
        <string>Display Options</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="dataflowPipelinesHorizontalLayout_">
          <item>
           <widget class="QLabel" name="pipelinesLabel_">
            <property name="text">
             <string>Dataflow Pipelines:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QRadioButton" name="manhattanPipesRadioButton_">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="whatsThis">
             <string>Switch to classic dataflow pipelines.</string>
            </property>
            <property name="text">
             <string>Manhattan</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="cubicPipesRadioButton_">
            <property name="whatsThis">
             <string>Switch to curved dataflow pipelines.</string>
            </property>
            <property name="text">
             <string>Cubic BÃ©zier</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="euclideanPipesRadioButton_">
            <property name="whatsThis">
             <string>Switch to straight, direct dataflow pipelines.</string>
            </property>
            <property name="text">
             <string>Euclidean</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Default Note Position:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="defaultNotePositionComboBox_">
            <property name="currentIndex">
             <number>4</number>
            </property>
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Tooltip</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Top</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Left</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Right</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bottom</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="1">
           <widget class="QCheckBox" name="dockableModulesCheckBox_">
            <property name="text">
             <string>Enable module dock</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="modulesSnapToCheckBox_">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Modules snap to grid</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="portSizeEffectsCheckBox_">
            <property name="text">
             <string>Highlight ports on hover</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="actionTextIconCheckBox_">
            <property name="text">
             <string>Execute Button State</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="tristate">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="errorGraphicItemsCheckBox_">
            <property name="text">
             <string>Display errors inline</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>591</width>
        <height>418</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Paths</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="pathSettingsGroupBox_">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>SCIRun Path Settings</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>436</width>
            <height>316</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="pathSettingsGridLayout_">
           <item row="0" column="0">
            <widget class="QLabel" name="scirunDataLabel_">
             <property name="text">
              <string>SCIRun Data</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="scirunDataLineEdit_">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="scirunDataPushButton_">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Set...</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="scirunNetsLabel_">
             <property name="text">
              <string>SCIRun Nets</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="scirunNetsLineEdit_">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="scirunNetsPushButton_">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Set</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="userDataLabel_">
             <property name="text">
              <string>User Data</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="userDataLineEdit_">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="userDataPushButton_">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Set</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="addToPathButton_">
             <property name="text">
              <string>Add to Path...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QTextEdit" name="scirunDataPathTextEdit_">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="dataSetGroupBox_">
         <property name="title">
          <string>SCIRun Data Set*</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <widget class="QWidget" name="layoutWidget1">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>351</width>
            <height>48</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="dataSetHorizontalLayout_">
           <item>
            <widget class="QLabel" name="dataSetLabel_">
             <property name="text">
              <string>Data Set</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dataSetLineEdit_">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="dataSetPushButton_">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Set</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="toolBoxPage2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>591</width>
        <height>418</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="moduleErrorDialogDisableCheckbox_">
         <property name="text">
          <string>Disable Module Error dialogs</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoModuleNoteCheckbox_">
         <property name="text">
          <string>Add helpful module notes automatically</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="highDPIAdjustCheckBox_">
         <property name="text">
          <string>Adjust for high-DPI displays (SCIRun restart required)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
